Assets/Scripts/Battle/BattleSystem/BattleParticipant.cs

```diff
@@ -129,16 +131,28 @@ public void AddStatus(Status status)
         Status newStatus = Instantiate<Status>(status, transform);
         _statuses.Add(newStatus);
         newStatus.AddStatus(this);
+        if (StatusPanel != null)
```

Comment:
A null value here indicates that we messed up when hooking up the UI, so we would want an error to display. Remove all these null checks.